pom.xml

```diff
@@ -4041,69 +4043,6 @@
                                 </execution>
                             </executions>
                         </plugin>
-                        <plugin>
```

Comment:
we need these goals to be included since the cache requires the clean goal but including them is painless because the cache takes care of most of the cost